why3/src/declaration.rs

```diff
@@ -56,6 +56,7 @@ pub struct Contract {
     pub requires: Vec<Exp>,
     pub ensures: Vec<Exp>,
     pub variant: Vec<Exp>,
+    pub may_panic: Option<Exp>,
```

Comment:
I'd like this to be called `raises` instead, I know Creusot leaks into the `why3` crate but that's something i'd like to incrementally clean up.